src/create-render-context.js

```diff
@@ -168,7 +173,12 @@ const createRenderContext = function(
     sandbox.global = sandbox;
     sandbox.self = sandbox;
 
-    // This makes sure that qTip2 doesn't try to use the canvas.
+    /**
+     * This makes sure that qTip2 doesn't try to use the canvas.
+     *
+     * TODO(somewhatabstact): Do we need this hack still? Is there are better
```

Comment:
I do believe this is still needed as we're still using qTip2. Here are some of the places:

```
javascript/add-child-package/with-hover-tip.jsx
javascript/checklist-package/learn-for-yourself.jsx
javascript/checklist-package/tooltip-action.jsx
javascript/clarifications-package/mod-controls.jsx
javascript/hover-card-package/hover-card.js
javascript/profile-package/widget-privacy-view.jsx
javascript/settings-package/settings.js
javascript/tasks-secondary-package/video-task-status.jsx
```

And some of those are used in other places too, I think.

Reply:
I wonder if it's something that the webapp code should be handling though, instead of the render server

Reply:
I've raised a ticket to cover this: [WEB-1393](https://khanacademy.atlassian.net/browse/WEB-1393)